app/controllers/api/v1/courses/enrollments_controller.rb

```diff
@@ -3,41 +3,37 @@ module V1
     module Courses
       class EnrollmentsController < ApplicationController
         expose :course
-        expose :enrollment, parent: :course
-        expose :enrollments, fetch: :fetch_enrollments
-
-        def index
-        end
+        expose :enrollment, -> { Enrollment.find_by(enrollment_params) }
```

Comment:
I do not understand, why should we find enrollments by course_id and user_id? Could you explain this use-case, please? And remove specific enrollment by id?
I see `has_many :enrollments, dependent: :destroy` association in the Course class. Maybe we should return all enrollments in the index method?

Reply:
I thought there could be only one enrollment of user to a course
`/courses/:course_id/enrollment` totally matches that. (In routes it is a single resource).

But if we going to add all enrollments endpoint in future,
we can create another controller on `/enrollments`.

But I don't like route like this `/courses/:course_id/enrollments/:id`
because it seems that there can be more than one enrollment to a course.

The other option was to create UNnested route `/enrollments/:id`, but the current option
seems more simple to me from the point of API consumer's view.

```
GET /courses/:id/enrollment
POST /courses/:id/enrollment
DELETE /courses/:id/enrollment
```